### PR TITLE
Set the genesis block parameters

### PR DESCRIPTION
**Pull request by WeiLoy**

### CODE REVIEW

**Review by cheng762 — APPROVED** [review]

**Review by niuxiaojie81 — APPROVED** [review]

**Comment by benbaley** [comment]
rerun tests

**Comment by benbaley** [comment]
rerun tests

**Review by fksyuan — APPROVED** [review]

**Comment by benbaley** [comment]
rerun tests

**Review by jiangshangqifeng — COMMENTED** [review]
LGTM

File: core/genesis.go [hunk 1]
```diff
@@ -492,16 +492,20 @@ func GenesisBlockForTesting(db ethdb.Database, addr common.Address, balance *big
 // DefaultGenesisBlock returns the PlatON main net genesis block.
 func DefaultGenesisBlock() *Genesis {
 
-	generalAddr := common.MustBech32ToAddress("lat1dl93r6fr022ca5yjqe6cgkg06er9pyqfwkwwdg")
+	generalAddr := common.MustBech32ToAddress("lat1sy6kxgpfgx7axrl86a368mj6r6fnagctqem69g")
 	generalBalance, _ := new(big.Int).SetString("9727638019000000000000000000", 10)
 
 	rewardMgrPoolIssue, _ := new(big.Int).SetString("200000000000000000000000000", 10)
 
+	manifesto := `"τῆς ἄνω ὁδοῦ ἀεὶ ἑξόμεθα καὶ δικαιοσύνην μετὰ φρονήσεως παντὶ τρόπῳ ἐπιτηδεύσομεν."
```
Comment (jiangshangqifeng) [inline core/genesis.go:500]: LGTM

**Pull request merged**